test-file.txt

```diff
@@ -1,19 +1,16 @@
 1.
-2.
 3.
 4.
 5.
-6.
+6. Line 2 was deleted, so this is now line 5
```

Comment:
2. Here's a comment on the addition of new text to the line previously known as 6. My comment is on the green part, not the red part.